ROADMAP.md

```diff
@@ -0,0 +1,191 @@
+
+Testground was used successfully at Protocol Labs
+
+- TODO: note on libp2p interop
+- TODO: note on filecooin
+- TODO: note on IPFS v0.5.0 (massive DHT and Bitswap improvements)
+- TODO: note on libp2p (gossipsub 1.1 security extensions)
+
+## Vision
+
+Testground As A Service embodies our long-term vision.
+
+- A single, scalable platform that one or more organizations can use,
+- The ability to track the impact of a change in terms of stability & performance across multiple projects,
+- The ability to experiment with large-scale networks and simplify the integration testing of libraries used across deep stacks.
+
+Products with similar ideas:
+
+- [CockroadDB performance tracker](https://cockroachdb.github.io/pebble/?max=local),
+- [Webkit Performance Dashboard](https://perf.webkit.org/v3/)
+
+## Problems we focus on
+
+We focus on reliability, usefulness, and sustainability. We want to ensure Testground is valuable and stable before we grow its feature set.
+
+### Table of Content
+
+- [1. Testground provides reliable results](#1-testground-provides-reliable-results)
+- [2. Testground can be set up as an organization-wide Service](#2-testground-can-be-set-up-as-an-organization-wide-service)
+- [3. The knowledge required to work efficiently with Testground is available and easy to access](#3-the-knowledge-required-to-work-efficiently-with-testground-is-available-and-easy-to-access)
+- [4. Testground Development follows high-standard](#4-testground-development-follows-high-standard)
+- [5. Testground provides the networking tooling required to test complex Distributed / Decentralized Applications](#5-testground-provides-the-networking-tooling-required-to-test-complex-distributed--decentralized-applications)
+- [6. Testground provides the tooling to make test maintenance & finding issues simple](#6-testground-provides-the-tooling-to-make-test-maintenance--finding-issues-simple)
+- [7. Testground covers every /essential/combination of languages and libraries required by its users](#7-testground-covers-every-essentialcombination-of-languages-and-libraries-required-by-its-users)
+
+### 1. Testground provides reliable results
+
+**Why:** an unreliable testing platform is just a noise machine. We need to secure our users' trust. Testground maintainers need clear feedback about stability improvements & regressions.
+
+#### Milestone 1: We have a stability metrics dashboard
+
+Maintainers and users have a way to measure and follow Testground's Stability over any "relevant" axis.
+
+This might combine different languages, runners (k8s, docker, local), and context (developer env, CI env, k8s env).
+
+#### Milestone 2: We have identified and reached our user's stability requirements. We resolved "most" of them
+
+- local runner,
+- docker runner,
+- EKS runner,
+
+#### Milestone 3: We have a performances dashboard
+
+Maintainers and users have a way to measure and follow Testground's Performances over any "relevant" axis.
```

Comment:
What would that be?

Reply:
That would be the first task: define these axes.